rootston/output.c

```diff
@@ -240,6 +241,11 @@ void output_add_notify(struct wl_listener *listener, void *data) {
 		wlr_output_layout_add_auto(desktop->layout, wlr_output);
 	}
 
+	if (roots_xcursor_theme_load(desktop->xcursor_theme, wlr_output->scale)) {
+		wlr_log(L_ERROR, "Cannot load xcursor theme with scale %d",
```

Comment:
This error message makes it sound like it failed because the scale is invalid while I think it can fail for other reasons.